src/group_theory/subgroup.lean

```diff
@@ -404,13 +406,11 @@ end
 @[to_additive] instance fintype_bot : fintype (⊥ : subgroup G) := ⟨{1},
 by {rintro ⟨x, ⟨hx⟩⟩, exact finset.mem_singleton_self _}⟩
 
-@[simp] lemma _root_.add_subgroup.card_bot : fintype.card (⊥ : add_subgroup A) = 1 :=
-fintype.card_eq_one_iff.2
-  ⟨⟨(0 : A), set.mem_singleton 0⟩, λ ⟨y, hy⟩, subtype.eq $ add_subgroup.mem_bot.1 hy⟩
-
--- `@[to_additive]` doesn't work, because it converts the `1 : ℕ` to `0`.
-@[simp] lemma card_bot : fintype.card (⊥ : subgroup G) = 1 :=
-fintype.card_eq_one_iff.2
+/-! curly brackets `{}` are used here instead of instance brackets `[]` because
```

Comment:
Whoops
```suggestion
/-- curly brackets `{}` are used here instead of instance brackets `[]` because
```

Reply:
I was wondering why you did that. I thought it best not left as a docstring but just a comment in the file since it isn't really an explanation of the lemma, just a not very important implementation detail.

Reply:
@eric-wieser :arrow_up:

Reply:
The curly vs square brackets are visible to readers of the lemma in doc-gen, so I think it's valuable for this to be visible. At any rate, the `-!` that's currently there is wrong, as that doesn't attach the comment to the lemma.

Reply:
Should it be a module docstring just above the lemma statement?

Reply:
That's what it currently is, but I think it should probably just be a lemma docstring - hence the suggestion to change from `!-` to `--`

Reply:
I think a normal comment would be fine. Users probably don't need to see this on hover. Seeing the comment when you look at the source will be helpful.